synth_machine/machine.py

```diff
@@ -107,7 +106,8 @@ def __init__(
             thresholds=(safety_thresholds if safety_thresholds else SAFETY_DEFAULTS),
         )
         self.buffer = {}
-        self.store = store
+        self.store = ObjectStore(storage_path, storage_options)
```

Comment:
Likely better to pass class to object rather than build it, not just for ease of use but it also makes the library more flexiable